history-of-the-quakers-v2/original/04-book-3-chapter-10.adoc

```diff
@@ -505,7 +513,7 @@ we will inform you that you may tell what you know in time,
 for if you be racked you will be a miserable man,
 and must have your head cut off besides.
 But I told them I had done no evil that I knew of, nor had any in my heart against them.
-Then he read a few lines to this purpose, we Leopold, Emperor,
+Then he read a few lines to this purpose, Leopold, Emperor,
```

Comment:
I don't understand the grammar, but it looks like the `we` should be retained, I think let's put it back:

![2020-05-18_12-05-57](https://user-images.githubusercontent.com/7050938/82234991-264f4380-9900-11ea-8dc3-ac01d35ebf07.png)

I only noticed that because when I was prepping the document I saw the same thing and almost removed it too :)

Reply:
Ahhh okay, I was wondering...in my original version it looks a little different, almost like it's a typo because it's attached to Leopold  like this: [weLeopold] so I figured, after much deliberation, it didn't belong! But I didn't even think about just adding a space, I searched for "weLeopold" like it was part of his name! :)